codegen/java-gen/src/test/resources/tests/action-service/simple-new-style/generated-unmanaged/org/example/service/MyServiceAction.java

```diff
@@ -6,6 +6,7 @@
 import org.external.ExternalDomain;
 
 // This class was initially generated based on the .proto definition by Akka Serverless tooling.
+// This is the implementation for the Action Service described in your example-action.proto file.
```

Comment:
Full path to proto file now showing here and below? Something missing?

Reply:
this example, as all the rest but one, is in the base src/main/proto folder. Only the path underneath, if any, appears. 
Does this make sense?

Reply:
have a look at here please
https://github.com/lightbend/akkaserverless-java-sdk/pull/820/files/315e6d01ea0c4a95ef7bcfc04626296d63fbf0c1#diff-2e0727ef0ef5a9a962fe8366da4c1ffe4b37e76ef5f6303f93898f66beda3cb7